src/pymail

```diff
@@ -84,72 +103,88 @@ def email_recv(use_sys_argv):
-        
-    # Get the latest email
-    result, data = mail.fetch(email_id, "(RFC822)")
-    raw_email = data[0][1]
+    return (_email, _pass)
 
-    print(raw_email)
 
 def login():
-    """Get account credentials"""
+    """Get account credentials from files or from user
+    input if the file specified does not exist.
+    """
+
+    # Define global variables
     global email_addr
     global email_pass
-    if os.path.isfile(install_path + "mailaddr") == True:
-        mail_addr_file = open(install_path + "mailaddr", 'r')
-        email_addr = mail_addr_file.read()
-        mail_pass_file = open(install_path + "mailpass", 'r')
-        email_pass = mail_pass_file.read()
+
+    # If the file specified exists
+    if os.path.isfile(install_path + "mailaddr"):
+    	# Get the variables from the two files; install_path + mail(addr/pass)
+    	email_addr, email_pass = [open(install_path + 'mail%s' % x).read().close() for x in ['addr', 'pass']]
     else:
-        email_addr = raw_input("Email address = ")
-        print("Email password = "),
-        os.popen("stty -echo")
-        email_pass = raw_input()
-        os.popen("stty echo")
-        print("\n")
+        email_addr, email_pass = get_email_pass()
+
 
 def config():
-    """Get and save account credentials"""
+    """Get the credentials from the user and store them
+    in install_path + mail(addr/pass).
+    """
+
+    # TODO: Implement encryption.
     print("BE AWARE THAT YOUR CREDENTIALS WILL NOT BE ENCRYPTED.")
-    email_addr = raw_input("Email address = ")
 
-    print("Email password = "),
-    os.popen("stty -echo")
-    email_pass = raw_input()
-    os.popen("stty echo")
-    print("\n")
+    email_addr, email_pass = get_email_pass()
 
-    mail_addr_file = open("mailaddr", 'w')
-    mail_pass_file = open("mailpass", 'w')
+    # List comprehensions are awesome
+    # Pretty self-explanatory, but saves the email and pass to the files
+    [open(install_path + 'mail%s' % x, 'w').write(y).close() for x, y in [['addr', email_addr], ['pass', email_pass]]]
 
-    mail_addr_file.write(email_addr)
-    mail_pass_file.write(email_pass)
 
 def pymail_help():
-    """Prints basic help"""
-    print("Usage:")
-    print("pymail help")
-    print("pymail config")
-    print("pymail send")
-    print("pymail send <receiver> <subject> <body>")
-    print("pymail receive")
-    print("pymail receive <n>")
-    print("pymail file <file> <delete_concept (y/n)>")
+    """Iterates over a list of options and prints them to the screen.
+    No list comp. here because print in python 2.x is a statement, not a function.
+    """
+
+    print('Usage:')
+
+    print('\n'.join(['pymail ' + x for x in['help', 'config', 'send', 'send <receiver> <subject> <body>',
+                                            'receive', 'receive <n>', 'file <file> <delete (y/n)>']]))
+
     print("For more help, see README.md")
 
 # -------------------------------------------------------
 
-# init
+# Initialization
```

Comment:
Some may say I'm not fun at parties. I say my attendance is making the social gathering more pleasurable